hooks/pre/40-el8_modules.rb

```diff
@@ -0,0 +1,14 @@
+if module_enabled?('katello') && el8?
+  execute!('dnf -y module enable pki-core')
+end
+
+if local_postgresql? && el8?
+  postgres_version, success = execute_command('postgres --version', false, true)
+
+  if success && !postgres_version.include?('12.')
```

Comment:
PG 13 is also out and I wouldn't be surprised if someone used it at some point. They may be annoyed that it forces them to downgrade.

Another thing to consider is users who use packages from postgresql.org. https://www.postgresql.org/download/linux/redhat/ suggests it installs as postgresql$VERSION-server (so postgesql12-server). Not sure if they'll show up and if so, how but we have seen community users who use them.

Reply:
Hrmm, sounds like you are saying I should just limit this to Katello and leave Foreman alone since it can be in a wide variety of states.

Reply:
I recall that user of postgresql.org was running Katello and ran into problems because of the evr package.

Reply:
Externally or local?

Reply:
https://community.theforeman.org/t/foreman-1-19-test-week/10533/4

Reply:
That was quite a bit ago. And I feel like Katello has traditionally supported PG that comes with the OS to reduce support burden and matrix. We did this on EL7 through the use of the SCL as well. I have simplified this PR to just do this for the Katello scenario.